aep/general/0003/aep.md

```diff
@@ -0,0 +1,112 @@
+# Glossary
+
+This AEP defines common terminology.
+
+## Guidance
+
+The following terminology **must** be used consistently throughout AEPs.
+
+### API
+
+Application Programming Interface. This can be a local interface (such as an
+SDK) or a Network API (defined below).
+
+### API Backend
+
+A set of servers and related infrastructure that implements the business logic
+for an API Service.
+
+### API Client
+
+An API Client is a program or library that perform a specific tasks by calling
+an API or generic tools, such as CLIs, that expose the API in a user-accessible
+fashion or operate on resource data at rest.
+
+Examples of clients include the following:
+
+- Command line interfaces
+- Libraries, such as an SDK for a particular programming language
+- Scripts that operates on a JSON representation of a resource after reading it
+  from an API
+- Tools, such as a [Declarative client][]
+- Visual UIs, such as a web application
+
+### API Definition
+
+A well-structured representation of an API Service.
+
+### API Endpoint
+
+Refers to a network address that an API Service uses to handle incoming API
+Requests. One API Service may have multiple API Service Endpoints, such as
+`https://pubsub.googleapis.com` and `https://content-pubsub.googleapis.com`.
```

Comment:
```suggestion
`https://pubsub.example.com` and `https://content-pubsub.example.com`.
```

Also, what about the `https://apis.example.com/pubsub` form?

Reply:
de-googling I'm all for! I'm a bit mixed about the "adding a sub-api via the path", because it pollutes the resource name. I'm open to discussion though.

Reply:
Well...what should an organization do if they don't have a domain dedicated to APIs?

For example, at Roblox, we have "apis.roblox.com".  We talked about getting a different domain or creating further subdomains (e.g. pubsub.apis.roblox.com) but both options got rejected (by people not organizationally local to the API platform).

Reply:
sure, I think that's a good point. Here's an issue to discuss: https://github.com/aep-dev/aep.dev/issues/74.